app/src/main/java/com/chess/clock/fragments/TimeSettingsFragment.java

```diff
@@ -172,30 +146,18 @@ void loadTheme(AppTheme theme) {
         if (adapter != null) {
             adapter.updateTheme(theme);
         }
-        if (adapterCAB != null) {
-            adapterCAB.updateTheme(theme);
-        }
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
             plusImg.setImageTintList(theme.primaryColorAsStateList(getContext()));
         }
     }
 
-    @Override
-    public void onDestroyView() {
-        super.onDestroyView();
-        if (mMultiSelectionController != null && mMultiSelectionController.isActionModeActive()) {
-            multiSelectionFinishedByOnDestroyView = true;
-            mMultiSelectionController.finish();
-        }
-        mMultiSelectionController = null;
-    }
-
     @Override
     public void onCreateOptionsMenu(@NonNull Menu menu, @NonNull MenuInflater inflater) {
         super.onCreateOptionsMenu(menu, inflater);
         inflater.inflate(R.menu.settings_actions, menu);
     }
 
+    @SuppressLint("NonConstantResourceId")
```

Comment:
I don't think we should suppress that (even though it's preexisting code). "Replace switch with if" option in Android studio would be better.  https://sites.google.com/a/android.com/tools/tips/non-constant-fields